Snake/SnakeTest/GameSnake_Test.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SnakeTest
+{
+    [TestClass]
+    public class GameSnake_Test
+    {
+        [TestMethod]
+
+        public void First_Snake_Created_at_Correct_Location()

```

Comment:
name is ok but could be better.
no need for all the capital letters.
 "Correct location" does not give a clue as to what it is. 
how about First_snake_created_at_start_location